api-tests/src/utils/jwt-decoder.ts

```diff
@@ -0,0 +1,12 @@
+import * as jose from "jose";
+import { VcJwtPayload } from "../types/external-api.js";
+
+export const decodeCredentialJwts = (jwts: string[]) => {
+  const mappedJwts: Record<string, VcJwtPayload> = {};
+  jwts.forEach((jwt) => {
+    const claims = jose.decodeJwt(jwt) as VcJwtPayload;
+    mappedJwts[claims.iss] = claims;
+  });
```

Comment:
If we were using Node 21+ we could use a groupBy here :(

`return jwts.map(jose.decodeJwt).groupBy((vc) => vc.iss)`

Reply:
:')